Dockerfile

```diff
@@ -1,3 +1,16 @@
+# Build ostreeuploader, aka fiopush/fiocheck
+FROM ubuntu:20.04
+RUN apt-get update
+RUN apt-get install -y wget git gcc make -y
+RUN wget -P /tmp https://go.dev/dl/go1.18.linux-amd64.tar.gz && \
+    tar -C /usr/local -xzf /tmp/go1.18.linux-amd64.tar.gz
```

Comment:
Do you need go 1.18? Just to understand when we should update this one in the future.

Another option is to enable https://launchpad.net/~longsleep/+archive/ubuntu/golang-backports and install go from there (which will always be the latest instead of having a fixed one).

```
sudo apt update
sudo apt-get install software-properties-common
sudo add-apt-repository ppa:longsleep/golang-backports
sudo apt update
sudo apt install golang-go
```

Reply:
> Do you need go 1.18? Just to understand when we should update this one in the future

It's safer to stick a specific version that the fiotools have been tested for, and the one that we use for `foundries/lmp-image-tools`, currently it's go1.18.1.

We may update golang version if the fiotools need it, if we do we update it in the both containers.

Reply:
Sure, works for me.